File: javascript/1038.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split(' ');
var code = parseInt(lines[0]);
var qunt = parseInt(lines[1]);
var value = 0;
switch(code) {
     case 1:value = 4;
          break;
     case 2:value = 4.5;
          break;
     case 3:value = 5;
          break;
     case 4:value = 2;
          break;
     case 5:value = 1.5;
          break;
}
var calculo = qunt * value
console.log("Total: R$ " + calculo.toFixed(2));

File: C++/1038.cpp
var input = require('fs').readFileSync('/dev/stdin', 'utf8')
var spl = input.trim().split("")
var r = " NO"

for (let i = 0; i <= spl.length - 1; i++) {
    if(spl[i] == 1 && spl[i+1] == 3){
        r = ""
        break
    }
}
console.log(spl.join("") + r + " es de Mala Suerte")